packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/__tests__/quanticSearchBoxInput.test.js

```diff
@@ -86,234 +86,266 @@ describe('c-quantic-search-box-input', () => {
-    it('should not display the submit button to the right of the searchbox', async () => {
-      const element = createTestComponent({
-        ...defaultOptions,
-        withoutSubmitButton: true,
-      });
-      await flushPromises();
-
-      const searchIcon = element.shadowRoot.querySelector(
-        selectors.searchBoxSearchIcon
-      );
-      const submitButton = element.shadowRoot.querySelector(
-        selectors.searchBoxSubmitBtn
-      );
-
-      expect(submitButton).toBeNull();
-      expect(searchIcon).not.toBeNull();
-      expect(searchIcon.classList.contains('slds-input__icon_left')).toBe(true);
-    });
-  });
-
-  describe('when the placeholder property receives a custom placeholder value', () => {
-    it('should display the custom value as the searchbox placeholder', async () => {
-      const customPlaceholder = 'Custom placeholder';
-      const element = createTestComponent({
-        ...defaultOptions,
-        placeholder: customPlaceholder,
-      });
-      await flushPromises();
-
-      const input = element.shadowRoot.querySelector(selectors.searchBoxInput);
-
-      expect(input.placeholder).toEqual(customPlaceholder);
-    });
-  });
-
-  describe('when suggestions are found', () => {
-    it('should display the suggestions in the suggestions list', async () => {
-      const element = createTestComponent({
-        ...defaultOptions,
-        suggestions: mockSuggestions,
-      });
-      await flushPromises();
-
-      const suggestionsList = element.shadowRoot.querySelector(
-        selectors.searchBoxSuggestionsList
-      );
-      expect(suggestionsList).not.toBeNull();
-
-      const suggestionsListItems =
-        suggestionsList.shadowRoot.querySelectorAll('li');
-      expect(suggestionsListItems).not.toBeNull();
-
-      expect(suggestionsListItems.length).toEqual(mockSuggestions.length);
-    });
-  });
-
-  describe('when focusing on the input', () => {
-    it('should dispatch a #quantic__showsuggestions custom event', async () => {
-      const element = createTestComponent();
-      setupEventListeners(element);
-      await flushPromises();
-
-      const input = element.shadowRoot.querySelector(selectors.searchBoxInput);
-      expect(input).not.toBeNull();
-
-      await input.focus();
-
-      expect(functionsMocks.exampleShowSuggestions).toHaveBeenCalledTimes(1);
-    });
-
-    describe('when selecting a suggestion from the suggestions list', () => {
-      it('should dispatch a #quantic__selectsuggestion event with the selected suggestion as payload', async () => {
+  [true, false].forEach((textareaValue) => {
+    describe(`when the textarea property is set to ${textareaValue}`, () => {
+      it(`should display the ${
+        textareaValue ? 'expandable' : 'default'
+      } search box properly`, async () => {
         const element = createTestComponent({
           ...defaultOptions,
-          suggestions: mockSuggestions,
+          textarea: true,
```

Comment:
```suggestion
          textarea: textareaValue,
```

shouldnt this be set to `textareaValue` ?

Reply:
Indeed, fixed!